output/adapter/evm.go

```diff
@@ -0,0 +1,77 @@
+package adapter
+
+import (
+	"context"
+	"log/slog"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/machinefi/w3bstream-mainnet/output/chain/eth"
+)
+
+// contractAbiJSON is the ABI of the contract
+// solidity interface: function setProof(string memory _proof) external;
+const (
+	contractMethod  = "setProof"
```

Comment:
maybe contractMethod & Abi is project config member

Reply:
Support for user-defined method name can be added in the future.